### PR TITLE
Add job feedback for employers

### PR DESCRIPTION
**Pull request by kendebacker**

### CODE REVIEW

File: src/components/job/JobView.jsx [hunk 1]
```diff
@@ -78,6 +79,12 @@ export default function JobView() {
       <div className="h-full w-full flex flex-col justify-between">
         <div className="flex flex-col card">
           <div className="card-body items-center">
+            {data.adminMessage && auth.user.role === ROLES.EMPLOYER ? (
+              <span>
+                <p className="font-bold">Administrator Feedback:</p>
+                <p>{data.adminMessage}</p>
+              </span>
+            ) : null}
```
Comment (makersmelx) [inline src/components/job/JobView.jsx:87]: guess should be one more condition: job is rejected

Comment (makersmelx) [reply src/components/job/JobView.jsx:87]: add style for it if you have time?

**Review by makersmelx — APPROVED** [review]

**Review by makersmelx — CHANGES_REQUESTED** [review]
see comments

File: src/components/job/JobView.jsx (hunk 1 above)
Comment (kendebacker) [reply src/components/job/JobView.jsx]: I added a status badge for style and only showing admin feedback if it's there

**Review by makersmelx — APPROVED** [review]

**Pull request merged**